client/app/lib/util/cleanup.coffee

```diff
@@ -3,16 +3,55 @@ kookies = require 'kookies'
 
 toBeDeleted = [
   'clientIPAddress' # an old cookie set by koding backend.
+  'kdproxy-preferred-domain' # old koding cookie
   'connect.sid'     # express cookie - infact this is session cookie, here for just in case.
-  'pnctest'         # set by PubNub for testing.
-  'hubspotutk'      # hubspot
-  'hsfirstvisit'    # hubspot
+  # 'pnctest'       # set by PubNub for testing. it is set on every load, so not necessary to remove.
   '__ssid'          # google map
+  'gsScrollPos'     # session cookie, wont be cleaned, not necessary
```

Comment:
aggreed it is too generic :) already created confusion but i am not the name father, "session cookie" is its attribute http://snpy.in/fZhdSY browsers cant alter that value, hence wont be cleaned